.prettierrc

```diff
@@ -1,8 +0,0 @@
-{
```

Comment:
Without a config file prettier wont auto-format on most IDEs (at least that's what I've experienced a lot).

Reply:
It was duplicated https://github.com/woodpecker-ci/plugin-ready-release-go/blob/main/.prettierrc.json

Reply:
Ohh missed that.